client/task-list/tasks/payments/RemotePayments/index.js

```diff
@@ -38,7 +39,7 @@ export const RemotePayments = ( { query } ) => {
 	const recommendedMethod = useMemo( () => {
 		const method = methods.find(
 			( m ) =>
-				( m.key === 'wcpay' && m.visible ) ||
+				( m.key === 'woocommerce_payments' && m.visible ) ||
```

Comment:
Heads up that these have been moved into a constant array `RECOMMENDED_GATEWAY_KEYS`.

Reply:
Cool, no worries here. Just updating references. Note that the logic/ui for WCPay is going to be different than other recommended gateways for all except the final stage (enabled + configured) so we may need more iteration here.